runtime/src/lib.rs

```diff
@@ -300,16 +354,18 @@ impl pallet_balances::Config for Runtime {
     type ReserveIdentifier = [u8; 8];
 }
 
+parameter_types! {
+	/// Relay Chain `TransactionByteFee` / 10
+	pub const TransactionByteFee: Balance = 10 * MICROUNIT;
+	pub const OperationalFeeMultiplier: u8 = 5;
```

Comment:
how was this determined?

Reply:
from here https://github.com/paritytech/polkadot/blob/v0.9.13/runtime/polkadot/src/lib.rs#L347-L349